build/stf-run-ci/tasks/create_catalog.yml

```diff
@@ -76,7 +76,7 @@
           dockerfile: |
             # The base image is expected to contain
             # /bin/opm (with a serve subcommand) and /bin/grpc_health_probe
-            FROM registry.redhat.io/openshift4/ose-operator-registry:v4.13
+            FROM {{default_operator_registry_image_base}}:{{default_operator_registry_image_tag}}
```

Comment:
Despite the name "default", they never get used to create a "final" variable and are used as-is everywhere else in the code.

Reply:
Some WIP on this in: https://github.com/infrawatch/service-telemetry-operator/pull/429/commits/cf2923eb3ee25081777da231be1a28effe1acc46

Reply:
I'm in favor of any of these solutions in order of preference.

1. Start breaking your PR into digestible pieces so we can merge the improvements to this code independently
   * This likely has other benefits, that PR is getting very long lived and far-ranging
2. Merge this and adjust your PR accordingly
3. Abandon this and just assume we won't care to change this value until after that PR merges

I won't bother to apply any additional pressure here since I'm not fixing a problem, just noticed this and thought to be proactive.

Reply:
Hi there. I'm back after 3 weeks and I can't find any other PR that contains this change or an equivalent/replacement/conflicting change.  The original link to https://github.com/infrawatch/service-telemetry-operator/commit/cf2923eb3ee25081777da231be1a28effe1acc46 shows this (due to force-push I assume?)

![image](https://github.com/infrawatch/service-telemetry-operator/assets/5091372/1cb47dca-b8d8-422b-b329-426128f00ae0)

I'm going to remove the do-not-merge label as I think this change can safely merge.

Reply:
You'll get the "this commit does not believe to any branch on this repository" generally because when we merge branches, it squashes all the commits into a single commit rather than performing a merge commit. I noticed that as well in the past.

In this case though, you might be correct, because that commit refers to a parent commit in https://github.com/infrawatch/service-telemetry-operator/pull/470 so yea... I don't know where it went either :)

I agree this can likely merge without issue.